Jenkinsfile

```diff
@@ -212,8 +217,20 @@ pipeline {
       parallel {
         stage ('Text Classification with BERT Test') {
           steps {
-            sh 'cd examples/nlp/text_classification && python text_classification_with_bert.py pl.trainer.gpus=1 model.language_model.pretrained_model_name=bert-base-uncased pl.trainer.max_epochs=1 model.language_model.max_seq_length=50 model.data_dir=/home/TestData/nlp/retail/ model.validation_ds.prefix=dev model.train_ds.batch_size=10 model.train_ds.num_samples=-1 model.language_model.do_lower_case=true'
-            sh 'rm -rf examples/nlp/text_classification/outputs'
+            sh 'cd examples/nlp/text_classification && \
+            python text_classification_with_bert.py \
+            model.language_model.pretrained_model_name=bert-base-uncased \
+            model.language_model.max_seq_length=50 \
+            model.data_dir=/home/TestData/nlp/retail/ \
+            model.validation_ds.prefix=dev \
+            model.train_ds.batch_size=10 \
+            model.train_ds.use_cache=false \
+            model.language_model.do_lower_case=true \
+            pl.trainer.gpus=[0] \
+            +pl.trainer.fast_dev_run=true \
+            exp_manager.root_dir=exp_bert_base_uncased \
+            '
```

Comment:
Should we set exp_manager to null for ci? Why do you want to log these experiments? You also need to add model.validation_ds.use_cache=false for squad tests. What about moving these args to model level to set only once?

Reply:
I think it is fine to not set exp_manager to null for CI. But then don't forget to cleanup (like @ericharper  does below to avoid collisions). This way exp_manager would be tested too. But no hard preference on this - I think it is fine to have it either as null or not null.

Reply:
This PR isn't about organizing NLP configuration, let's save that for a different PR.

I think it's good to have exp_manager in the test to make sure it's working. No one cares about the logs themselves, which is why we remove them right away.